src/app/core/provide-core.ts

```diff
@@ -176,7 +176,6 @@ export const models =
     ResearcherProfile,
     OrcidQueue,
     OrcidHistory,
-    AccessStatusObject,
```

Comment:
Why was this necessary?

IIRC changes like these could cause some unexpected issues when data services get used _slightly_ differently down the line, all `@typedObject` models should remain declared here

Reply:
We already included the AccessStatusObject a couple lines above. I removed it here as it was inserted twice.

Reply:
Perfect!